### DISCUSSION TITLE
Unverified account by Railway

### DISCUSSION THREAD

**Discussion by Faolad** [community/community#57892] [New to GitHub]
Hi i tried to connect my github account with railway but i got message that that my account is not verified and that "the attached GitHub account does not provide enough data on who you are.

**Comment by queenofcorgis** [comment]
Hi there @Faolad, 

For the first bit - I'd head to Settings --> Emails and check that your email has been verified, if not you'll have the option to resend the verification email.  For the second part, that seems to be up to Railway's discretion, I'd recommend asking them what data is missing and what they need. 

Good luck!
